### PR TITLE
Cleaning up go.mod and adding command

### PR DESCRIPTION
**Pull request by jonathan-buttner**
Not sure why the statik.go file changed again 🤔  is it because the go.mod and go.sum files changes?

### CODE REVIEW

**Review by mtojek — COMMENTED** [review]
Hm... isn't it because of the changelog.yml which is included in statik.go?

**Comment by jonathan-buttner** [comment]
> Hm... isn't it because of the changelog.yml which is included in statik.go?

Oh yeah that could be.

File: versions/1/changelog.yml [hunk 1]
```diff
@@ -27,7 +27,7 @@
   - description: Add `hidden` fields to data stream manifest file.
     type: enhancement
     link: https://github.com/elastic/package-spec/pull/91
-  - description: Revert: Allow `condition` under inputs.
+  - description: 'Revert: Allow `condition` under inputs.'
```
Comment (jonathan-buttner) [inline versions/1/changelog.yml:30]: This was complaining in vscode because `description: Revert:` is invalid yml I think.

**Review by mtojek — APPROVED** [review]
Ship it!

**Pull request merged**